SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialLatencyTracer.h

```diff
@@ -96,6 +96,8 @@ class SPATIALGDK_API USpatialLatencyTracer : public UObject
 
 	bool IsValidKey(TraceKey Key);
 	TraceKey GetTraceKey(const UObject* Obj, const UFunction* Function);
+	TraceKey GetTraceKey(const UObject* Obj, const UProperty* Property);
+	TraceKey GetTraceKey(uint32 Entity, uint32 Component);
```

Comment:
Nit: Entity should be EntityId. Same for Component.

Do we not have access to Worker_EntityId?

Also a case of const args.